AutoGrade/admin.py

```diff
@@ -67,8 +67,6 @@ def __init__(self, *args, **kwargs):
             instructor = Instructor.objects.filter(user=self.current_user).first()
             self.fields['course'].queryset = Course.objects.filter(instructor=instructor)
 
-    list_filter = ('course', )
```

Comment:
Er ... I think this got killed in this commit. Need this in the file.

Reply:
list_filter were placed in AssignmentFormAdmin(forms.ModelForm) and AssignmentModelAdmin(admin.ModelAdmin). But list_filter have no meaning in forms.ModelForm instance. 

@recluze

Reply:
Oh, ok. Sorry about that.